enterprise/settings/test.py

```diff
@@ -170,6 +170,7 @@ def root(*args):
         'service_user': SERVICE_USER_THROTTLE_RATE,
     },
     'DATETIME_FORMAT': '%Y-%m-%dT%H:%M:%SZ',
+    'TEST_REQUEST_DEFAULT_FORMAT': 'json',
```

Comment:
Here's another bonus change that allows all of our client.post() calls in unit tests to be cleaner.

Reply:
backing out this change since it had the side effect of making PUT requests use JSON when they shouldn't.